metaworld/policies/sawyer_reach_v1_policy.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy
+
+
+class SawyerReachV1Policy(Policy):
+    def __init__(self):
+        super().__init__(5.)
+
+    @staticmethod
+    def parse_obs(obs):
+        return {
+            'hand_xyz': obs[:3],
+            'puck_xyz': obs[3:-3],
+            'goal_vec': obs[-3:]
+        }
```

Comment:
perhaps you can assert that `sum([len(x) for x in retval.values()]) == len(obs)`?

Reply:
you want the assert at the unpacking site, not in a random test.

asserts are for defining local invariants